src/components/vanilla/Description.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { SMALL_FONT_SIZE } from '../constants';
+
+export default function Title({ description }: { title?: string }) {
+  return (
+    !!description && (
```

Comment:
suggestion - to make it more readable use ifs

`if(!description) { return null }

return .....`